src/supertux/menu/sorted_contrib_menu.cpp

```diff
@@ -17,15 +18,18 @@
 #include "supertux/menu/sorted_contrib_menu.hpp"
 
 #include <sstream>
-#include "util/gettext.hpp"
+
+#include "boost/format.hpp"
+#include "gui/menu_manager.hpp"
```

Comment:
One small detail: There should be an empty line between headers from dependencies (SDL, Physfs, Boost, etc.) and headers from the source directory